src/Story/StoryMetaData.php

```diff
@@ -154,8 +155,43 @@ private function parseDate (string $date) : \DateTimeImmutable
 		return $parsed;
 	}
 
+	/**
+	 */
 	public function getPosition () : ?int
 	{
 		return $this->data["position"] ?? null;
 	}
+
+	/**
+	 * @return array<array{id: int, name: string, slug: string, published: bool, full_slug: string, is_folder: bool, parent_id: int}>
+	 */
+	public function getAlternateLanguages () : array
+	{
+		return $this->data["alternates"] ?? [];
+	}
+
+	/**
+	 * Returns the mapping of locale to full slug for alternative translated versions of this story.
+	 *
+	 * @return array<string, string> locale => full_slug
+	 */
+	public function getTranslatedDocumentsMapping () : array
+	{
+		$mapping = [];
+
+		foreach ($this->getAlternateLanguages() as $alternateLanguage)
+		{
+			$slug = $alternateLanguage["full_slug"];
+			$locale = \mb_substr($slug, 0, \strpos($slug, "/") ?: null);
```

Comment:
Should we store the locale as its own field within the story?

Reply:
We don’t get the full story here but just some metadata. 

Or do you mean to move the parsing one level up into the other getter?

Reply:
Yes, just the parsing

Reply:
Understood. Is updated 👍